app/components/chat.tsx

```
@@ -43,117 +202,311 @@ export function Chat(props: { repo: string }) {
+
+        if (json.result.paths.length === 0) {
+            toast({
+                title: `No path found`,
+                description: `no path found between node ${path.start.name} - ${path.end.name}`,
+            })
+            return
+        }
+
+        const formattedPaths: { nodes: any[], edges: any[] }[] = json.result.paths.map((p: any) => ({ nodes: p.filter((node: any, i: number) => i % 2 === 0), edges: p.filter((edge: any, i: number) => i % 2 !== 0) }))
+        chart.add(formattedPaths.flatMap((p: any) => graph.extend(p, false, path)))
+        formattedPaths.forEach(p => p.edges.forEach(e => e.id = `_${e.id}`))
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)